app/javascript/components/util/InfoTooltip.js

```diff
@@ -203,8 +203,8 @@ const TOOLTIP_TEXT = {
   ),
   extendedIsolation: (
     <div>
-      Used by the system to determine eligibility to appear on the <i>Records Requiring Review</i> line list. A case cannot appear on the{' '}
-      <i>Records Requiring Review</i> line list until after the user-specified date. This field may be blank.
+      Used by the system to determine eligibility to appear on the<i>Records Requiring Review</i>line list. A case cannot appear on the{' '}
+      <i>Records Requiring Review</i>line list until after the user-specified date. This field may be blank.
```

Comment:
Removing this space actually messes up the tooltip, can we revert this?
<img width="255" alt="Screen Shot 2020-10-22 at 5 03 32 PM" src="https://user-images.githubusercontent.com/11698457/96929692-86867a80-1488-11eb-95ab-7d7ceee374e7.png">